src/poetry/inspection/lazy_wheel.py

```diff
@@ -383,8 +383,10 @@ def _content_length_from_head(self) -> int:
         :raises HTTPRangeRequestUnsupported: if the response fails to indicate support
                                              for "bytes" ranges."""
         self._request_count += 1
-        head = self._session.head(self._url, headers=self._uncached_headers())
+        head = self._session.head(self._url, headers=self._uncached_headers(), allow_redirects=True)
         head.raise_for_status()
+        # Redirects may change this
```

Comment:
does that matter?  is this necessary?  won't we just get another redirect next time and follow that too?

Reply:
I agree, not required for this change.